ledger/src/entry.rs

```diff
@@ -819,23 +795,6 @@ mod tests {
         assert!(e0.verify(&zero));
     }
 
-    #[test]
-    fn test_witness_reorder_attack() {
```

Comment:
@garious , I didn't understand this unit test to be demonstrating anything different than `test_transaction_reorder_attack()` (ie. swapping two transactions that are identical other than instruction data). Am I missing something?

Reply:
Yep, looks redundant